src/banner/Banner.style.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components'
+
+import { color, componentSizes, space } from '../_utils/branding'
+import { TextDisplay1 } from '../typography/display1'
+import { TextTitle } from '../typography/title'
+
+export const StyledBanner = styled.section`
+  background-color: ${color.boostBlue};
+  padding: ${space.xxl} ${space.xl};
+  height: 19.625rem; //314px magic number
+  overflow: hidden;
+`
+
+const Heading = styled(TextDisplay1)`
+  margin-bottom: ${space.xl};
+`
+const Paragraph = styled(TextTitle)`
+  margin-bottom: ${space.xl};
+`
+const Actions = styled.div`
+  margin-top: ${space.xl};
+
+  > button:first-child {
+    margin-right: ${space.l};
+  }
+`
+
+export const Frame = styled.div`
+  padding: ${space.l} ${space.xxl} 0 ${space.xxl};
+  background-image: url('./phone-frame.svg');
```

Comment:
That's frustrating, I can't get the image to be displayed this way using css. Might need some config twisting out of my knowledge